src/Modules/Guilds/Hooks/types.ts

```diff
@@ -0,0 +1,32 @@
+export type WagmiUseContractReadResponse<T> = Pick<

```

Comment:
Nice! Can't believe `wagmi` hasn't fixed their types better imho. This could almost be open-sourced.